ql/src/java/org/apache/hadoop/hive/ql/exec/ExportTask.java

```diff
@@ -53,9 +53,7 @@ public int execute() {
       work.acidPostProcess(db);
       TableExport tableExport = new TableExport(exportPaths, work.getTableSpec(),
           work.getReplicationSpec(), db, null, conf, work.getMmContext());
-      if (!tableExport.write()) {
```

Comment:
Why is the check removed?

Reply:
Have refactored the code and same behavior is achieved in TableExport.export() now